docs/topics/native/native-objc-interop.md

```diff
@@ -16,6 +16,25 @@ Kotlin modules can be used in Swift/Objective-C code if compiled into a
 framework ([see here for how to declare binaries](multiplatform-build-native-binaries.md#declare-binaries)).
 See [Kotlin Multiplatform Mobile Sample](https://github.com/Kotlin/kmm-basic-sample) for an example.
 
+### Hiding Kotlin declarations
+
+You can choose not to export Kotlin declaration to Objective-C and Swift with special annotations:
```

Comment:
If you don't want to export Kotlin declarations to Objc and Swift, use special annotations:

WDYT?